### ISSUE TITLE
Trophy Garden entries missing from Generation IV

### ISSUE THREAD

**Issue by Hyreon** [PokeAPI/pokeapi#696]
In Generation IV, diamond pearl and platinum, talking to Backlot in the Pokemon mansion enables the appearance of certain Pokemon in the Trophy Garden.

The API currently does not include these encounters, so Jigglypuff has no encounters for diamond and pearl.

My current project uses this API to calculate which games are required for which Pokemon. If this feature were not added, then I would have to manually override the API's response for this Pokemon, and potentially others.

**Comment by Machine-Maker** [comment]
Yeah, I will do this. I have a plan for it already, just doing my changes one-by-one to avoid having to constantly fix conflicts.
